src/LSPEncoder.cpp

```diff
@@ -96,7 +111,7 @@ void	LSPEncoder::AddLSPInstrument(int id, int modInstrument, int sampleOffsetInB
 	// add sample to the LSP bank
 	assert(unsigned(id) < LSP_INSTRUMENT_MAX);
 	assert((modInstrument >= 1) && (modInstrument <= 31));
-	
+
```

Comment:
there is plenty of single line change or TAB change like this. Is that expected?

Reply:
I think that is just CLion cleaning up unneeded whitespace.